packages/bastet_lwt/bastet_lwt.0.1.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "Lwt implementations for bastet"
+maintainer: ["me@risto.codes"]
+authors: ["Risto Stevcev"]
+license: "BSD-3-Clause"
+tags: ["category theory" "abstract algebra" "algebra" "cats" "lwt"]
+homepage: "https://github.com/Risto-Stevcev/bastet-lwt"
+doc: "https://risto-stevcev.github.io/bastet-lwt"
+bug-reports: "https://github.com/Risto-Stevcev/bastet-lwt/issues"
+depends: [
+  "ocaml" {>= "4.06.1"}
+  "bastet" {>= "1.2.4"}
+  "lwt" {>= "5.2.0"}
+  "alcotest" {>= "1.0.1" & with-test}
+  "alcotest-lwt" {= "1.0.1" & with-test}
+  "merlin" {>= "3.3.3" & with-test}
+  "ocamlformat" {>= "0.13.0" & with-test}
```

Comment:
Same question as in https://github.com/ocaml/opam-repository/pull/16082/files#r398871612